bakerydemo/base/models.py

```diff
@@ -283,6 +321,14 @@ class GalleryPage(BasePageFieldsMixin, Page):
 
 
 class FormField(AbstractFormField):
+    """
+    Wagtailforms is a module to introduce simple forms on a Wagtail site. It
+    isn't intended as a replacement to Django's form support but as a quick way
+    to generate general purpose data-collection form without having to write
+    code. We use it on the site for a contact form. You can read more about
+    Wagtail forms at:
+    http://docs.wagtail.io/en/v1.9/reference/contrib/forms/index.html
```

Comment:
better to use 'latest' rather than a specific version, I think:

http://docs.wagtail.io/en/latest/reference/contrib/forms/index.html